tests/processors/merge_trials_identifiers/test_processor.py

```diff
@@ -31,7 +29,6 @@ def test_updates_trial_with_records_identifiers(self, conn, trial, record):
         updated_trial = conn['database']['trials'].find_one(id=trial)
 
         assert updated_trial['identifiers'] == record_attrs['identifiers']
-        assert updated_trial['updated_at'].replace(tzinfo=None) > trial_last_updated
```

Comment:
Just to make sure I understood, the issue is that the code you've wrote to dump the database using SQLAlchemy doesn't copy the triggers, right?

It's tricky. In principle, the test database must be the same as the production database, otherwise we risk hitting hard (or impossible) to reproduce bugs. I went ahead and changed it to use regular postgres tools on https://github.com/opentrials/processors/pull/106. This simplified the code and surfaced other issues with SQLAlchemy not copying constraints. Please review that and then we can rebase it into this master before merging it in, so this test will continue working.